examples/approx_planted_point.jl

```diff
@@ -1,3 +1,12 @@
+
+# This script tests different methods for solving optimization problems involving approximate planted points
```

Comment:
I really like the addition of an introductory comment on the top of each example, but we should be a bit more concise for each of them. Anything that can be read from the code is not necessary. For instance 
> checks the solution against expected values, and verifies the correctness of  the results using `@test` assertions.